qt/applications/workbench/workbench/widgets/settings/general/test/test_general_settings.py

```diff
@@ -7,12 +7,7 @@
 #  This file is part of the mantidqt package
 from __future__ import absolute_import, unicode_literals
 
-import sys
-
-if sys.version_info.major >= 3:
```

Comment:
I think we need to keep the version checks. For Python 2 `mock` comes from a 3rd party package but for Python 3 it is in the std library. We keep doing this over the code base so it's probably about time we created an `import_mock` function. Perhaps we could put this in the `mantid.py3compat` module?

Reply:
I am happy to do this, but we also install `python-mock` and `python3-mock` packages which makes the whole thing kind of redundant

Reply:
I'd like to remove those packages when we move the Python 3 as they should become obselete.